### ISSUE TITLE
Uncaught missing library : week0

### ISSUE THREAD

**Issue by trayfellow** [ShadowMario/FNF-PsychEngine#4]
i was compiling on html5 and when i entered the character editor menu, it froze and i checked the logs it said "uncaught missing library : week0", ill try to compile on desktop and see if the same thing happens

**Comment by trayfellow** [comment]
update : it works fine on desktop, also another issue, placing notes on html5 crashes the game

**Comment by ShadowMario** [comment]
Did you change anything on the week?

**Comment by trayfellow** [comment]
> Did you change anything on the week?

yeah, also this time i tested a html5 build on a fresh unmodified source code of the version i was using, and it looks like character customizer works fine, the only thing that crashes the game now is when you place a note in chart menu

**Comment by ShadowMario** [comment]
Check out if it still crashes on the latest update.

**Comment by trayfellow** [comment]
placing notes in html5 still crashes
![image](https://user-images.githubusercontent.com/48515908/130383133-5a229146-38b7-491c-a15f-4ad3be1ca9e9.png)

**Comment by ShadowMario** [comment]
It's not about placing notes, it seems to be related to selecting notes, i'm still trying to figure out what's wrong with it, please check out if the Character Editor is working now.

**Comment by AbnormalPoof** [comment]
character editor works, going to the note tab freezes

**Comment by trayfellow** [comment]
ok so just to clear up the character customizer issue, its fixed now (hopefully cause it works with my modded character now), so the only thing that needs to be fixed now is how placing notes in the charting menu crashes your game on html5

**Comment by trayfellow** [comment]
another update : ok nvm while the character customizer loads on html5 when i switch to a different character like bf it crashes too
